tests/e2e/tidbcluster/serial.go

```diff
@@ -153,197 +150,6 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 		})
 	})
 
-	ginkgo.Context("[Feature: Defaulting and Validating]", func() {
-		var ocfg *tests.OperatorConfig
-		var oa tests.OperatorActions
-
-		ginkgo.BeforeEach(func() {
```

Comment:
In my view, there is unnecessary to add e2e test to cover default and validate templates. Unit test is enought.